Mathlib/MeasureTheory/Function/SimpleFuncDenseLp.lean

```diff
@@ -437,255 +437,254 @@ end SimpleFunc
 
 namespace Lp
 
-open AeEqFun
+open AEEqFun
 
 variable [MeasurableSpace α] [NormedAddCommGroup E] [NormedAddCommGroup F] (p : ℝ≥0∞)
   (μ : Measure α)
 
 variable (E)
 
-/-- `Lp.simple_func` is a subspace of Lp consisting of equivalence classes of an integrable simple
+/-- `Lp.simpleFunc` is a subspace of Lp consisting of equivalence classes of an integrable simple
     function. -/
-def simpleFunc : AddSubgroup (lp E p μ) where
-  carrier := { f : lp E p μ | ∃ s : α →ₛ E, (AEEqFun.mk s s.AEStronglyMeasurable : α →ₘ[μ] E) = f }
+def simpleFunc : AddSubgroup (Lp E p μ) where
+  carrier := { f : Lp E p μ | ∃ s : α →ₛ E, (AEEqFun.mk s s.aestronglyMeasurable : α →ₘ[μ] E) = f }
   zero_mem' := ⟨0, rfl⟩
-  add_mem' := fun f g ⟨s, hs⟩ ⟨t, ht⟩ =>
-    ⟨s + t, by
-      simp only [← hs, ← ht, ae_eq_fun.mk_add_mk, AddSubgroup.coe_add, ae_eq_fun.mk_eq_mk,
-        simple_func.coe_add]⟩
-  neg_mem' := fun f ⟨s, hs⟩ =>
-    ⟨-s, by
-      simp only [← hs, ae_eq_fun.neg_mk, simple_func.coe_neg, ae_eq_fun.mk_eq_mk,
-        AddSubgroup.coe_neg]⟩
-#align measure_theory.Lp.simple_func MeasureTheory.lp.simpleFunc
+  add_mem' := by
+    rintro f g ⟨s, hs⟩ ⟨t, ht⟩
+    use s + t
+    simp only [← hs, ← ht, AEEqFun.mk_add_mk, AddSubgroup.coe_add, AEEqFun.mk_eq_mk,
+      SimpleFunc.coe_add]
+  neg_mem' := by
+    rintro f ⟨s, hs⟩
+    use -s
+    simp only [← hs, AEEqFun.neg_mk, SimpleFunc.coe_neg, AEEqFun.mk_eq_mk, AddSubgroup.coe_neg]
+#align measure_theory.Lp.simple_func MeasureTheory.Lp.simpleFunc
```

Comment:
Why was this rewritten? Should this get a porting note?

Reply:
"unknown free variable '_uniq.546677'". I'll make a note after this PR is merged